ClientCore/LoadingScreenController.cs

```diff
@@ -5,18 +5,19 @@ namespace ClientCore
 {
     public static class LoadingScreenController
     {
+        private static readonly Random _random = new Random();
+        
         public static string GetLoadScreenName(string sideId)
         {
             int resHeight = UserINISettings.Instance.IngameScreenHeight;
-            int randomInt = new Random().Next(1, 1 + ClientConfiguration.Instance.LoadingScreenCount);
-            string resolutionText;
-
-            if (resHeight < 480)
-                resolutionText = "400";
-            else if (resHeight < 600)
-                resolutionText = "480";
-            else
-                resolutionText = "600";
+            int randomInt = _random.Next(1, 1 + ClientConfiguration.Instance.LoadingScreenCount);
+            
+            string resolutionText = resHeight switch
+            {
+                < 480 => "400",
+                < 600 => "480",
+                _ => "600"
+            };
```

Comment:
Honestly we are tired of such a change. If you have modified surrounding codes, you can change it but otherwise, I suggest not doing so to reduce possible pains in git rebase/cherrypicks etc.